pom.xml

```diff
@@ -18,13 +18,12 @@
     specific language governing permissions and limitations
     under the License.
 
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+--><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
```

Comment:
I guess that if there are conflicts with the Apache parent pom it may indicate compliance issue on the Apex side. Will it be better to resolve it, rather than going unnoticed?

Do we need to include LICENSE and NOTICE into apex jars? There is no consistency among Apache projects, but it seems that Apache prefers it to be included.